presto-tests/src/main/java/com/facebook/presto/tests/QueryAssertions.java

```diff
@@ -332,37 +332,90 @@ public static void copyTpchTables(
             Session session,
             Iterable<TpchTable<?>> tables)
     {
-        copyTpchTables(queryRunner, sourceCatalog, sourceSchema, session, tables, false);
+        log.info("Loading data from %s.%s...", sourceCatalog, sourceSchema);
```

Comment:
There are many other connectors that call this method using TpchTable table object instead of table names. I find it unnecessary to change all the callers using our newly defined method call since current testing implementation with 2 entrance points work fine.

Reply:
@fgwang7w `Iterables.transform(tables, table -> table.getTableName())` would convert the tables parameter from `Iterable<TpchTable<?>>` to `Iterable<String>`. You will keep the original function `copyTpchTables` but change it to call copyTables. And all other existing callsites don't need to change.

```
public static void copyTpchTables(
            QueryRunner queryRunner,
            String sourceCatalog,
            String sourceSchema,
            Session session,
            Iterable<TpchTable<?>> tables)
    {
        copyTables(queryRunner, sourceCatalog, sourceSchema, session, Iterables.transform(tables, table -> table.getTableName()), false, false);
    }

    public static void copyTables(
            QueryRunner queryRunner,
            String sourceCatalog,
            String sourceSchema,
            Session session,
            Iterable<String> tables,
            boolean ifNotExists,
            boolean bucketed)
    {
        log.info("Loading data from %s.%s...", sourceCatalog, sourceSchema);
        long startTime = System.nanoTime();
        for (String table : tables) {
            copyTable(queryRunner, sourceCatalog, sourceSchema, session, table, ifNotExists, bucketed);
        }
        log.info("Loading from %s.%s complete in %s", sourceCatalog, sourceSchema, nanosSince(startTime).toString(SECONDS));
    }
```

Reply:
got it, thank you so much!